boards/arm/dragonclaw/doc/index.rst

```diff
@@ -0,0 +1,50 @@
+.. _dragonclaw_board:
+
+Google Dragonclaw Development Board
+###################################
+
+Overview
+********
+
+Dragonclaw is a board created by Google for fingerprint-related functionality
+development. See the `Dragonclaw Schematics`_ for board schematics, layout and
+BOM.
+
+Board has connectors for fingerprint sensors. Console is exposed over μServo
+connector. MCU can be flashed using μServo or SWD.
+
+Hardware
+********
+
+- STM32F412CGU6 UFQFPN48 package
+
+Peripherial Mapping
+===================
+
+- USART_1 TX/RX : PA9/PA10
+- USART_2 TX/RX : PA2/PA3
+- SPI_1 CS/CLK/MISO/MOSI : PA4/PA5/PA6/PA7
+- SPI_2 CS/CLK/MISO/MOSI : PB12/PB13/PB14/PB15
+
+Programming and Debugging
+*************************
+
+Build application as usual for the ``dragonclaw`` board, and flash
+using μServo or SWD. If μServo is used, please follow the `Chromium EC Flashing
```

Comment:
Instead of SWD you could say something like "using an external J-Link connected to J4.

Reply:
Done